code/modules/mining/equipment/mining_tools.dm

```diff
@@ -117,6 +117,20 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	custom_materials = list(/datum/material/iron=12050) //metal needed for a crowbar and for a knife, why the FUCK does a knife cost 6 metal sheets while a crowbar costs 0.025 sheets? shit makes no sense fuck this
 
+/obj/item/pickaxe/emergency
```

Comment:
There already exists `/obj/item/pickaxe/emergency`. It's the emergency disembarkation tool. `code/modules/shuttle/emergency.dm`.

Reply:
SpacemanDMM should lint for duplicate definitions like this, honestly. Oh well.

Reply:
It's used intentionally throughout the codebase for better or for worse